<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1385</width>
    <height>715</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Linde Button Manufacturing</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-10</y>
      <width>1311</width>
      <height>711</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Experimento</string>
     </attribute>
     <widget class="QWidget" name="widgetProgramming" native="true">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>-80</y>
        <width>800</width>
        <height>651</height>
       </rect>
      </property>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>800</width>
         <height>64</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonProgram">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>130</y>
         <width>211</width>
         <height>64</height>
        </rect>
       </property>
       <property name="text">
        <string>Iniciar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_Programming">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>800</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Prepare la muestra y presion el botón &quot;Iniciar&quot;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="rowSelectExperiment" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>800</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Seleccione experimento</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboExperiments"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="buttonStop">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>210</y>
         <width>211</width>
         <height>64</height>
        </rect>
       </property>
       <property name="text">
        <string>Detener</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="signalWidget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>290</y>
         <width>801</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>Configuración</string>
     </attribute>
     <widget class="QWidget" name="widgetSetup" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>421</width>
        <height>441</height>
       </rect>
      </property>
      <widget class="QLabel" name="labelUsedPort">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>981</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>401</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonAdd">
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonModify">
          <property name="text">
           <string>Modificar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelete">
          <property name="text">
           <string>Eliminar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListView" name="listExperiments">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>401</width>
         <height>341</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonTerminate">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>221</width>
      <height>64</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Terminar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
